pkg/util/callback.go

```diff
@@ -39,6 +39,10 @@ func (c *callbackInvoker) ExecuteCallback(callbackURL string, success bool, mess
 		"success": success,
 	}
 
+	for name, value := range labels {
+		d[name] = value
+	}
```

Comment:
Is it necessary to keep the json that we send flat? Why not add a labels map under the main map?